pkg/process/init.go

```diff
@@ -227,7 +227,7 @@ func (p *Init) ExitStatus() int {
 	return p.status
 }
 
-// ExitedAt at time when the process exited
```

Comment:
it is function name. we can not change that.

Reply:
OK I Will change it